docs/UserGuide.md

```diff
@@ -1063,6 +1065,16 @@ A: It is currently not possible to export your data to another software.
 
 A: HomeBoss only supports English characters.
 
+**Q: What happens if someone accesses the json file (file where the Customer/Delivery data is kept) and edits the 
+data to be invalid? **
+
+A: In the rare event that this occurs, if the Customer data is the one that is edited and becomes invalid, all the 
+Customer data will be wiped completely and even the unique IDs of the Customers will start from 1. (even a clear 
+command will not cause the unique IDs to start from 1) The Delivery data,however, will still be intact.
```

Comment:
I believe the Delivery Data would get wiped if the Customer Data is invalid since delivery references customer